api/.env.test

```diff
@@ -3,3 +3,5 @@ KERNEL_CLASS='App\Kernel'
 APP_SECRET='$ecretf0rt3st'
 SYMFONY_DEPRECATIONS_HELPER=999999
 PANTHER_APP_ENV=panther
+PANTHER_ERROR_SCREENSHOT_DIR=./var/error-screenshots
+TRUSTED_HOSTS=^localhost|api|h2-proxy|example\.com$
```

Comment:
Missing parentheses: `^(localhost|api|h2-proxy|example\.com)$`
Also in existing `api/.env` line 26